src/views/Search/Records.vue

```diff
@@ -16,33 +21,74 @@
      */
     export default {
         name: "Records",
-        data(){
+        data() {
             return {
-                valid_request: this.is_request_valid()
+                /* Be careful, in the current system registry and type are reversed, which is why we have registry bound
+                to type and type bound to registry in the above object.
+                */
+                recordTypes: {
+                    Standards: "Standard",
+                    Databases: "Database",
+                    Policies: "Policy",
+                    Collections: "Collection"
+                },
+                pagination: {
+                    page: 1,
+                    perPage: 30
+                },
+                content: [],
+                client: null,
+                searchString: ""
             }
         },
         computed: {
-            currentPath: function(){
-                return this.$route.path.replace('/', '');
+            currentPath: function () {
+                const title =  this.$route.path.replace('/', '');
+                return title.charAt(0).toUpperCase() + title.slice(1);
             }
         },
         components: {
-            OutputTable
+            OutputGrid
         },
         methods: {
-            /**
-             * Is the request valid
-             * @param {Object} request_body - the request itself
-             * @returns {boolean}
+            /** This methods get the data from the client depending on the current page.
+             * @returns {Promise}
              */
-            is_request_valid(request_body){
-                if (request_body) return true;
-                return false;
+            getData: async function () {
+                if (this.client){
+                    // This is data coming from the JSON query
+                    if (!searchRecords.hasOwnProperty("queryParam") || searchRecords.queryParam === null){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Do not access Object.prototype method 'hasOwnProperty' from target object.](https://app.codacy.com/gh/FAIRsharing/fairsharing.github.io/pullRequest?prid=4893952)